snippets/visualbasic/VS_Snippets_Wpf/timingbehaviors_procedural_snip/visualbasic/sampleviewer.xaml.vb

```diff
@@ -35,19 +35,19 @@ Namespace Microsoft.Samples.Animation.TimingBehaviors
 			Get
 				Return theClock
 			End Get
-			Set(ByVal value As Clock)
-				If theClock IsNot Nothing Then
-					RemoveHandler theClock.CurrentTimeInvalidated, AddressOf onClockTimeInvalidated
+            Set(value As Clock)
```

Comment:
Indentation is a problem here (actually, a problem throughout the file)

Reply:
I think the indenting of above is correct in current commit.